### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tmcgilchrist**
Wrong link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This library is an attempt to implement a _simple_ but type-safe alternative.
 
 ## Example
 
-See [Basic.hs](blob/master/example/webship-example/src/Webship/Example/Basic.hs).
+See [Basic.hs](https://github.com/charleso/webship/blob/master/webship-example/src/Webship/Example/Basic.hs).
```
Comment (charleso) [inline README.md:22]: Hmm, pity it can't be relative?

**Comment by charleso** [comment]
:+1: 

You have push rights. :)

**Comment by charleso** [comment]
I really wanted to put an example in the `README` but I know I would forget to update it if the API changes. Thoughts?

**Comment by tmcgilchrist** [comment]
I agree you really want it to compile the examples. Could you do that via haddock comments? (not that it will compile them there either) Is literate haskell still a thing you could use?

**Pull request merged**